xenial/2.3.6/Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM ubuntu:xenial
 
-MAINTAINER Daniel Alan Miller <dalanmiller@rethinkdb.com>
+MAINTAINER Gabor Boros <gabor.brs@gmail.com>
```

Comment:
Maintainer is deprecated: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Although we don't actively recommend using labels (see linked issue for reasoning), they are available for adding data like maintainer. Image label names should adhere to the image spec: https://github.com/opencontainers/image-spec/blob/v1.0.1/annotations.md.

\- https://github.com/docker-library/official-images/issues/3540#issuecomment-530925319

Reply:
Since the Maintainer is deprecated and the labels are not actively recommended (and for me it is not that important to have my name there), I removed the maintainer. 😊